spec/classes/patroni_spec.rb

```diff
@@ -328,6 +328,15 @@
         end
       end
 
+      context 'tags => true' do
```

Comment:
There should be tests for all potential changes here.

What if 

tags = true and tags_parameters is empty?
tags = true and tags_parameters has values?
tags = false and tags_parameters is empty?
tags = false and tags_parameters has values?